### ISSUE TITLE
Tooling: cannot use path aliases in imports

### ISSUE THREAD

**Issue by samijaber** [BuilderIO/mitosis#524]
Even though the typescript server accepts them just fine in my VSCode, the build fails if we use a path alias.

https://github.com/BuilderIO/mitosis/blob/b8d083ca78bdaec332745d2a0f8c0890335072c4/packages/core/src/generators/vue.ts#L3-L16

These all have to be relative paths. If I were to write:

```ts
 import { collectCss } from 'src/helpers/collect-styles'; 
```

Then the build crashes, even though it seems to be properly setup in `tsconfig.json`. It's probably some minor configuration issue! This is a great good-first-issue for newcomers

**Comment by samijaber** [comment]
I think this impacts both `core` and our `cli` projects, but I am not entirely sure.

**Comment by decadef20** [comment]
I'll look into this.

**Comment by decadef20** [comment]
I find a related issue [tsc - doesn't compile alias paths](https://stackoverflow.com/questions/59179787/tsc-doesnt-compile-alias-paths).
Because tsc don't compile alias path, if we set an alias in `core`,  other projects in `packages` and `example` which use `core` also need to set alias.

If we set it in `example` like this:
```js
"paths": {
      "@builder.io/mitosis": ["../../packages/core/src"],
      "core/*": ["../../packages/core/src/*"]
 }
```
which compiles failed.
